src/booking/components/booking-navigation/booking-navigation.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { BookingNavigationComponent } from './booking-navigation.component';
+
+describe('BookingNavigationComponent', () => {
+  let component: BookingNavigationComponent;
+  let fixture: ComponentFixture<BookingNavigationComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ BookingNavigationComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(BookingNavigationComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
Any chance of testing anything else on this component?

Reply:
I've had a look at the code and other examples where similar navigation is being used. The component currently has 95.65% coverage and I'd only be adding superfluous tests to see if a state is in a list of states or if a trigger is emitted when I call the trigger method.